### ISSUE TITLE
The big bar on top that callout the merge only appear on some pages

### ISSUE THREAD

**Issue by nloureiro** [ethereum/ethereum-org-website#7002]
**Describe the bug**

The big bar on top that callout the merge only appear on some pages

**Expected behavior**

Should be on all pages? or if in just a few, should be consistent

**Screenshots**

![Screen Shot 2022-07-11 01 08 14 PM](https://user-images.githubusercontent.com/1120748/178261099-9506edef-d0e4-45af-9ef7-db476f27514c.png)
![Screen Shot 2022-07-11 01 07 15 PM](https://user-images.githubusercontent.com/1120748/178261105-b23ffa20-4cfc-4152-87bd-bf27555b7c2f.png)
![Screen Shot 2022-07-11 01 07 05 PM](https://user-images.githubusercontent.com/1120748/178261112-2e383c4a-d9ba-466a-bd58-3738753b05ba.png)
![Screen Shot 2022-07-11 01 06 57 PM](https://user-images.githubusercontent.com/1120748/178261115-6b7780bf-22aa-4bdd-ac24-64ddc08d448f.png)

**Additional context**

pages with top
Homepage + MD pages (not sure if all) + some pages in the developers section (here it's not consistent)

**Comment by samajammin** [comment]
This seems to me a good opportunity to address our banner & callout usage across the site & to feed this into the design system. e.g. when should we use this banner component? Should the UI be different for some callout that is page specific ("this content is out of date") vs. applicable to the whole site ("the merge is coming, please update your nodes").

**Comment by minimalsm** [comment]
> This seems to me a good opportunity to address our banner & callout usage across the site & to feed this into the design system. e.g. when should we use this banner component? Should the UI be different for some callout that is page specific ("this content is out of date") vs. applicable to the whole site ("the merge is coming, please update your nodes").

Agree! Probably a discussion for the design system but I don't think our current callouts are very attention-grabbing if this is important information. Perhaps a good solution would be to have types of callout (i.e. normal, warning, alert). [Heres a blog post with multiple types of callouts](https://kentcdodds.com/blog/how-to-use-react-context-effectively).

**Comment by nloureiro** [comment]
> > This seems to me a good opportunity to address our banner & callout usage across the site & to feed this into the design system. e.g. when should we use this banner component? Should the UI be different for some callout that is page specific ("this content is out of date") vs. applicable to the whole site ("the merge is coming, please update your nodes").
> 
> Agree! Probably a discussion for the design system but I don't think our current callouts are very attention-grabbing if this is important information. Perhaps a good solution would be to have types of callout (i.e. normal, warning, alert). [Heres a blog post with multiple types of callouts](https://kentcdodds.com/blog/how-to-use-react-context-effectively).

Yes total agree. We probably will split the types of call outs.

 - for this issue I was referring to top bar, probably something related to the header component and will have a effect site wide
 - For in content call outs we will o other issue because will have implications within the content. probably a isolated component that can be used in the content that can have an attribute for: **warning** _or_ **alert** _or_ **stand_out**

both will be designed within the design system EPIC before coding.

For this issue my idea was if we want to have a quick fix to those screenshots or wait for the fina design system solution.

**Comment by corwintines** [comment]
Given the merge is done, I think this is pretty stale 😅 

Going to close this out, and we can address banners in another issue.
